spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/feign/support/SpringMvcContract.java

```diff
@@ -144,6 +154,25 @@ public MethodMetadata parseAndValidateMetadata(Class<?> targetType, Method metho
 		return md;
 	}
 
+	protected void parseRegexAndUpdatePath(MethodMetadata md, Method method) {
+		if (isAnnotated(method, RequestMapping.class)) {
+			String url = md.template().url();
+			int paramCount = method.getParameterTypes().length;
+			for (int i = 0; i < paramCount; i++) {
+				MethodParameter param = new SynthesizingMethodParameter(method, i);
+				param.initParameterNameDiscovery(parameterNameDiscoverer);
+				if (param.hasParameterAnnotation(PathVariable.class)) {
+					PathVariable pathVariable = param.getParameterAnnotation(PathVariable.class);
```

Comment:
We went through steps to remove reference to `PathVariable` from this class. See `PathVariableParameterProcessor`.